js/UrbanAirship.js

```diff
@@ -402,6 +405,89 @@ class UrbanAirship {
   static isQuietTimeEnabled(): Promise<boolean> {
     return UrbanAirshipModule.isUserNotificationsEnabled();
   }
+
+  /**
+   * Sets the badge number for iOS.
+   *
+   * @param {number} badgeNumber specified badge to set.
+   */
+  static setBadgeNumber(badgeNumber: number) {
+    UrbanAirshipModule.setBadgeNumber(badgeNumber);
+  }
+
+  /**
+   * Gets the current badge number for iOS.
+   *
+   * @return {Promise.<number>} A promise with the result.
+   */
+  static getBadgeNumber(): Promise<number> {
```

Comment:
You need to rebase this branch on master and remove the commit for your other PR